app/cdap/components/Administration/TetheringTabContent/reducer.ts

```diff
@@ -36,6 +36,11 @@ export const initialConnectionsState = {
   establishedConnections: [],
 };
 
+const apiConnectionActions = {
```

Comment:
Can this be defined in one place?

Reply:
I wanted to do that initially, but we don't keep constants at an appropriate place for these two files to access. There is one `constants.ts` file for tethering table, and one for new requests. Didn't make sense to put this in either. Also didn't make sense to create a constant file just for this one.